src/esp/assets/ResourceManager.cpp

```diff
@@ -2224,6 +2224,13 @@ gfx::PhongMaterialData::uptr ResourceManager::buildFlatShadedMaterialData(
   // flat materials
   auto finalMaterial = gfx::PhongMaterialData::create_unique();
 
+  // texture transform, if the material has a texture; if there's none the
+  // matrix is an identity
+  // TODO this check shouldn't be needed, remove when this no longer asserts
+  // in magnum for untextured materials
+  if (material.hasTexture())
+    finalMaterial->textureMatrix = material.textureMatrix();
+
```

Comment:
braces please, as per Meta C++ style guide :)

Reply:
Clang Tidy was not bullying me about this, so I thought I'm fine here. Always pushing the limits!